packages/blockchain-wallet-v4-frontend/src/scenes/Signup/components/SofiSignupForm/index.tsx

```diff
@@ -79,19 +86,16 @@ const SofiSignupForm = (props: Props) => {
     }
   }) as SofiMigrationStatusResponseType
 
-  const { country, email, state } = sofiJwtPayload
+  const { country, state } = sofiJwtPayload
   const passwordValue = formValues?.password || ''
   const isUSStateUnsupported = bakktRedirectUSStates.includes(formValues?.state)
-
-  const dataGoal = goals.find((g) => g.name === 'signup')
+  // @ts-ignore
```

Comment:
hate this but could not figure out the TS error. will hopefully clean this up later this week